static/sched/distribits2024.xml

```diff
@@ -689,27 +689,27 @@
     <event guid="44622511-c5c2-4ca8-9978-fa1a153b8f80">
       <start>14:50</start>
       <duration>00:20</duration>
-      <title>TBA</title>
+      <title>Questions and panel discussion</title>
       <abstract>
-        TBA
+        Questions and panel discussion
       </abstract>
       <persons>
         <person></person>
       </persons>
     </event>
 
     <event guid="a81918df-a057-48b1-b016-ce31f59b02a3">
-      <start>15:10</start>
-      <duration>00:20</duration>
-      <title>Questions and panel discussion</title>
+      <start>15:40</start>
+      <duration>00:40</duration>
+      <title>Dataverse mini-symposium</title>
       <abstract>
-        Questions and panel discussion
+        TBA
       </abstract>
     </event>
```

Comment:
Would you like to add speakers at this point, or only when the abstract is announced?

```suggestion
      <persons>
        <person>Phil Durbin</person>
        <person>Jan Range</person>
        <person>Oliver Bertuch</person>
      </persons>
    </event>
```

Reply:
Later.